lua/entities/gmod_wire_expression2/core/e2lib.lua

```diff
@@ -86,7 +86,7 @@ local function maketype(typeid)
 	return typename or "unknown"
```

Comment:
Any idea how the "unknown" can be triggered?
The way I see it, it's impossible and thus should be removed along with the temporary variable.